modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/near/GridNearTxLocal.java

```diff
@@ -3327,14 +3233,7 @@ public IgniteTxMappings mappings() {
 
     /** {@inheritDoc} */
     @Override public boolean queryEnlisted() {
-        if (!txState.mvccEnabled())
-            return false;
-        else if (qryEnlisted)
-            return true;
-        else if (mappings.single())
-            return !mappings.empty() && mappings.singleMapping().queryUpdate();
-        else
-            return mappings.mappings().stream().anyMatch(GridDistributedTxMapping::queryUpdate);
+        return false;
```

Comment:
Why removed

`if (mappings.single())
            return !mappings.empty() && mappings.singleMapping().queryUpdate();
        else
            return mappings.mappings().stream().anyMatch(GridDistributedTxMapping::queryUpdate);` ?

Reply:
Because this code should be executed only when mvcc enabled